spec/acceptance/class_spec.rb

```diff
@@ -10,12 +10,10 @@
     pkg_cmd = 'yum info nginx | grep "^From repo"'
     pkg_remove_cmd = 'yum -y remove nginx nginx-filesystem passenger'
     pkg_match = case fact('operatingsystemmajrelease')
-                when '8'
-                  test_passenger = false
                 when '7' # https://blog.phusion.nl/2020/05/29/passenger-6-0-5/
```

Comment:
Hum, this release note says they added support for CentOS/RHEL 8. I guess we can have CI that test passenger on at least RHEL 8, but probably more changes are needed and that is another story for another PR..